src/tiledimage.js

```diff
@@ -674,6 +674,27 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
         this._setScale(height / this.normHeight, immediately);
     },
 
+    /**
+     * Sets an array of polygons to crop the TiledImage during draw tiles.
+     * The render function will use the default non-zero winding rule.
+     * @param Polygons represented in an array of pair array in pixels.
+     * Example format: [
+     *  [[197,172],[226,172],[226,198],[197,198]], // First polygon
+     *  [[328,200],[330,199],[332,201],[329,202]]  // Second polygon
+     * ]
+     */
+    setCroppingPolygons: function( polygons ) {
```

Comment:
I think for consistency we should be using OpenSeadragon.Point here instead of arrays. That said, we could make it so any object with an `x` and a `y` would work, so it's Point compatible, but you don't need to make Points if you don't want to.

Reply:
I have added support for point object.
Internally, it is convert to `xy` pair array, so functions depending on this doesn't need to change.